### PR TITLE
Tm refactor drink recipe

### LINKED ISSUE #72: Style error message and loading message on DrinkRecipe page view

### PR DESCRIPTION
**Pull request by tarynmartin**
### Type of change made:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features

### Detailed Description / What does this PR do?
- Add styling for error message and loading message
- Refactor code in DrinkRecipe for better accessibility
- Add styling for automani font to work with h tags

### Files modified:
- index.css, DrinkRecipe.js, DrinkRecipe.css

### Has This Been Tested?
- [ ] No
- [x] Yes

### Are there specific methods for testing?
- in terminal, run npm start and got to a recipe in browser
- to see loading message styling, change first conditional in render to force loading message
- to see error message styling, alter api call to force error message

### What are the relevant tickets?
resolves #72 

### Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/DrinkRecipe/DrinkRecipe.css [hunk 1]
```diff
@@ -8,6 +8,14 @@
   text-align: center;
 }
 
+.loading,
```
Comment (rwilliams659) [inline src/components/DrinkRecipe/DrinkRecipe.css:11]: Glad we've got this styled and looking pretty now!

File: src/components/DrinkRecipe/DrinkRecipe.js [hunk 2]
```diff
@@ -41,15 +41,12 @@ export class DrinkRecipe extends Component {
       let instructions = recipe.instructions.split('. ');
       return (
         <section className='drink-recipe'>
-          <Link className='back-btn' to='/'>
-            Back
-          </Link>
           <div className='drink-data'>
-            <img src={recipe.picture} className='drink-image' alt='glass of the drink' />
+            <img src={recipe.picture} className='drink-image' alt={`Glass of ${recipe.name}`} />
```
Comment (rwilliams659) [inline src/components/DrinkRecipe/DrinkRecipe.js:45]: Nice call on alt text here

**Review by rwilliams659 — APPROVED** [review]
Looks great!

File: src/components/DrinkRecipe/DrinkRecipe.js [hunk 3]
```diff
@@ -41,15 +41,12 @@ export class DrinkRecipe extends Component {
       let instructions = recipe.instructions.split('. ');
       return (
         <section className='drink-recipe'>
-          <Link className='back-btn' to='/'>
-            Back
-          </Link>
```
Comment (relyt4me) [inline src/components/DrinkRecipe/DrinkRecipe.js:46]: I actually think we should keep this

**Review by relyt4me — APPROVED** [review]

**Pull request merged**